src/main/java/Models/Prescription/PrescriptionDAO.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Models.Prescription;
+
+import Utils.Database;
+
+import java.sql.*;
+
+/**
+ *
+ * @author Bredan
+ */
+public class PrescriptionDAO {
+    
+    public static void insertPrescription(Prescription prescription){
+        Connection con = Database.getInstance().getConnection();
+        
+        try {
+            PreparedStatement ps = con.prepareStatement("INSERT INTO PRESCRIPTION (PATIENTID, MEDICINE, QUANTITY, REPEATING, DATE_ISSUED, END_DATE) VALUES (?, ?, ?, ?, ?, ?)");
+            ps.setInt(1, prescription.getPatientID());
+            ps.setString(2, prescription.getMedicine());
+            ps.setInt(3, prescription.getQuantity());
+            ps.setBoolean(4, prescription.getRepeating());
+            
+            java.sql.Date issueDate = java.sql.Date.valueOf(prescription.getIssueDate());
```

Comment:
See the above comments about the date types in the Prescription class.

Reply:
changed to date types